Linux Script/configure-linux.sh

```diff
@@ -875,7 +876,7 @@ searchAndFetch()
 {
 	url=$2
 
-	result=$(wget -qO- /dev/null --user "$LOGGLY_USERNAME" --password "$LOGGLY_PASSWORD" "$url")
+	result=$(curl -s -u $LOGGLY_USERNAME:$LOGGLY_PASSWORD $url)
```

Comment:
@mchaudhary and @mostlyjason, Here I have replaced the wget command wth curl. I looked in differents environments to check if curl is present or not and I found it installed in almost each distribution listed below

Amazon AMI 2017.03
Ubuntu 14
RedHat
SUSE
Fedora
CentOS

I have also tested the **configure-linux.sh** script with this updated code and everything looked fine to me.